demo/SPE10/steady.c

```diff
@@ -136,15 +142,13 @@ int main(int argc, char **argv) {
   ierr = DMSetFromOptions(dm); CHKERRQ(ierr);
   ierr = DMViewFromOptions(dm, NULL, "-dm_view"); CHKERRQ(ierr);
 
-  /* Setup problem parameters */
-  TDy  tdy;
-  ierr = TDyCreateWithDM(dm,&tdy); CHKERRQ(ierr);
-  ierr = TDySetDiscretizationMethod(tdy,WY); CHKERRQ(ierr);
-  ierr = TDySetFromOptions(tdy); CHKERRQ(ierr);
-  ierr = TDySetup(tdy); CHKERRQ(ierr);
   ierr = ReadSPE10Permeability(tdy,ang); CHKERRQ(ierr);
   ierr = TDySetDirichletValueFunction(tdy,Pressure,NULL); CHKERRQ(ierr);
 
+  /* Setup problem parameters */
+  ierr = TDySetupDiscretization(dm,tdy); CHKERRQ(ierr);
```

Comment:
Shouldn't this be `TDySetDM`?

Reply:
@ghammond86 I just saw [your comment](https://github.com/TDycores-Project/TDycore/pull/109#discussion_r508645420) about `TDySetDM -> TDySetupDiscretization`